### PR TITLE
Pipelines chart v1.0.33

### PR DESCRIPTION
**Pull request by rimusz**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/pipelines/values.yaml [hunk 1]
```diff
@@ -617,6 +625,9 @@ pipelines:
       defaultMinionInstanceSize: "c4.large"
       allowDynamicNodes: true
       allowCustomNodes: true
+      {{- range $key, $value := .Values.runtimeOverride }}
```
Comment (danielezer) [inline stable/pipelines/values.yaml]: Wouldn't it be easier here to just use `toYaml`?

Comment (rimusz) [reply stable/pipelines/values.yaml]: I copied what @khazina had done

File: stable/pipelines/templates/pipelines-hpa.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{{- if .Values.pipelines.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta1
```
Comment (eldada) [inline stable/pipelines/templates/pipelines-hpa.yaml]: Can we use `v1`?

File: stable/pipelines/values.yaml (hunk 1 above)
Comment (eldada) [reply stable/pipelines/values.yaml]: Should probably be `with` and `tpl`. We can change it later on...

File: stable/pipelines/templates/pipelines-hpa.yaml (hunk 2 above)
Comment (rimusz) [reply stable/pipelines/templates/pipelines-hpa.yaml]: I set the same version we have in Xray

**Pull request merged**